runelite-client/src/main/java/net/runelite/client/plugins/friendnotes/FriendNoteOverlay.java

```diff
@@ -49,6 +49,9 @@ private FriendNoteOverlay(Client client, FriendNotesPlugin plugin, TooltipManage
 		this.tooltipManager = tooltipManager;
 	}
 
+	/**
+	 * Add a friend note tooltip to a hovered friend list entry, if note exists.
```

Comment:
I don't think adding comments to overriden methods is very useful. I would just make this inline comment inside the method

Reply:
Isn't it common for overridden methods to have their own, more specific jdocs? I see this in Java's own standard library all the time.

Reply:
Maybe if anyone will ever reference the method directly. But this one is used only by overlay system in generic way, so it should not be here.

Reply:
Alright, I moved those jdocs onto their relevant lines.